src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netcoreapp1.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MvcRazorContentRoot Condition="'$(MvcRazorContentRoot)'==''">$(MSBuildProjectDirectory)</MvcRazorContentRoot>
+    <MvcRazorOutputPath Condition="'$(MvcRazorOutputPath)'==''">$(OutputPath)</MvcRazorOutputPath>
```

Comment:
As far as I can tell OutputPath is correct whether you're doing build or publish.